### ISSUE TITLE
Older versions of portable (zip, no-admin) vscode? Current ones are very slow!

### ISSUE THREAD

**Issue by sdlins** [Microsoft/vscode#60933]
I posted my question at Stackoverflow, but, there are already other question exactly like this over there since Feb 20, **2017** with no answer! So, it is definitely a *BUG* if you see by an user point view!

Please see it [here](https://stackoverflow.com/questions/42353255/how-to-download-older-version-of-vscode-portable) and mine [here](https://stackoverflow.com/questions/52810019/where-we-can-get-older-versions-of-vscode-like-1-25-in-zip-format).

I just cant see **why** MS dont use coherency in somethings. If a new VSCode is launched, it is made available in Source Code, Portable and Exe formats. But if we need an older version, only Source Code and Exe is available? Is there **any** logic on it?

Just to make it clear: I love VSCode. I just dont love MS. But I need it for Win64 because I have to use it in my work where I dont have admin privileges. (Linux at home).

Thank you.

**Comment by egamma** [comment]
Previous versions are available from the release notes, but these pages do not include the zip/archive versions, see below.

![image](https://user-images.githubusercontent.com/172399/46931845-3381bd80-d04d-11e8-9847-5eab2dcdfc5c.png)

**Comment by kAlvaro** [comment]
Is there a workaround? I even tried Wayback Machine without success: links expire, redirect to latest version or are not archived at all.

**Comment by octref** [comment]
As @egamma mentioned, you can go to the release notes page to download old release. One additional trick, you can find a link to download your platform's build, and replace the version number

- https://vscode-update.azurewebsites.net/1.27.2/win32-x64-user/stable
- https://vscode-update.azurewebsites.net/<your-version-here>/win32-x64-user/stable

**Comment by nancym** [comment]
I agree that this is a BUG. I have spent hours trying to install a version of VS Code that works when I'm offline because 1.28.1 does not. This offline BUG is known, e.g. see _Code fails to start/open when offline_ #7570. I installed the Insiders version 1.29 but that also does not work offline. On my other Windows 10 device I have 1.27.2 installed and that works when I'm offline. Please post here and in Issue #7570 explicit directions for how to get a version of VS Code that works offline. Thank you.

**Comment by octref** [comment]
@sdlins Why do you have to use an old version of VS Code though?

@egamma Should I add zip installer link to each release note? IMO it makes the listing a bit cluttered. I would prefer to have a old download listing page and make the download widget point to there, like so:

![image](https://user-images.githubusercontent.com/4033249/46982535-72d1fd80-d091-11e8-9dc6-50dcf8c9ff22.png)

**Comment by sdlins** [comment]
@octref because [this](https://github.com/Microsoft/vscode/issues/56644#issuecomment-429835123) that is related to [this](https://github.com/Microsoft/vscode/issues/60933#issuecomment-430040006).

**Comment by kAlvaro** [comment]
> As @egamma mentioned, you can go to the release notes page to download old release. One additional trick, you can find a link to download your platform's build, and replace the version number

@octref Unless I missed where exactly to look at, all those links point to installer versions. This ticket is about ZIP/Portable versions.

**Comment by egamma** [comment]
>@egamma Should I add zip installer link to each release note? IMO it makes the listing a bit cluttered. I would prefer to have a old download listing page and make the download widget point to there, like so:

@octref I agree this makes it cluttered and I like your proposal of a separate old download listing page.

**Comment by sdlins** [comment]
Hi guys, is there any way (even a complex one) or workaround to get the v1.25.1 (win10 64 zip) right now? I am really needing it. Thank you so much!

**Comment by Joelism** [comment]
@sdlins https://vscode-update.azurewebsites.net/1.25.1/win32-x64-archive/stable

**Comment by sdlins** [comment]
@Joelism thank you so much! :+1: You helped me a lot! (This issue can already be closed!)

Just kidding! lol

**Comment by cateyes99** [comment]
@flamott i'm looking for the zip package download link for **Windows 64 bit** `1.10.2` version. Following your way, I tried `https://vscode-update.azurewebsites.net/1.10.2/win32-x64-archive/stable`, it returned `Not Found`. What i found is your way works up to [`1.13.0`](https://update.code.visualstudio.com/1.13.0/win32-x64-archive/stable). Is there a way to find zip packages of Win 64 bit versions older than `1.13.0`?

**Comment by flamott** [comment]
@cateyes99 Windows 64 bit hit stable in [1.15](https://code.visualstudio.com/updates/v1_15#_windows-64-bit), maybe no 64 bit was built before 1.13?

**Comment by cateyes99** [comment]
@flamott interesting even the [`1.15`](https://code.visualstudio.com/updates/v1_15#_windows-64-bit) update note said so, but `1.13.0`'s 64 bit installer is also available. Anyway i found [`1.10.2`](https://vscode-update.azurewebsites.net/1.10.2/win32-archive/stable)'s 32 bit zip package for Windows. Thanks,

**Comment by RikkiGibson** [comment]
The artifacts are all there. It would be handy to just add the zip links on the release page. When I want to reinstall an old version of Code to verify a regression (i.e. it worked in the old version but not in the new version), it is really handy to use the Portable install option.

**Comment by dziku86** [comment]
What about x86/ia32 versions? Is it possible to download 32-bit archived version of insiders edition?

**Comment by binave** [comment]
run
```
wget --content-disposition https://go.microsoft.com/fwlink/?LinkID=620882 https://go.microsoft.com/fwlink/?LinkID=620884 https://go.microsoft.com/fwlink/?Linkid=850641

```
https://update.code.visualstudio.com/latest/linux-x64/stable
https://update.code.visualstudio.com/latest/darwin/stable
https://update.code.visualstudio.com/latest/win32-x64-archive/stable

replace `latest` version number, 
e.g.
wget --content-disposition https://update.code.visualstudio.com/1.56.2/darwin/stable
